ColorPatterns/Extensions/CGFloat+Converters.swift

```diff
@@ -10,6 +10,7 @@ import UIKit
 
 extension CGFloat {
 
+    // TODO: Replace with custom colors? How will I add 1000 custom colors?
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/vandermesis/ColorPatterns/blob/c07cd9a3b7a63e61e86332aeb4fa4cabf8b81c17/ColorPatterns/Extensions/CGFloat+Converters.swift#L13"></span>TODOs should be resolved (Replace with custom colors? Ho…).<br />
<code>todo</code> <code>CGFloat+Converters.swift:13</code></td>
    </tr>
  </tbody>
</table>